source/material_model/melt_simple.cc

```diff
@@ -742,8 +759,7 @@ namespace aspect
     void
     MeltSimple<dim>::create_additional_named_outputs (MaterialModel::MaterialModelOutputs<dim> &out) const
     {
-      if (this->get_parameters().use_operator_splitting
-          && out.template get_additional_output<ReactionRateOutputs<dim> >() == NULL)
+      if (this->get_parameters().use_operator_splitting && out.template get_additional_output<ReactionRateOutputs<dim> >() == NULL)
```

Comment:
do you still need the first condition?

Reply:
Yes. I first removed it, but it caused some kind of problem. I don't remember what exactly it was, but I think it might have been related to the point that we still allow this material to be used without operator splitting if melt transport is switched off (in this case, there are no reactions, so it doesn't matter), and I got some error message in one of the tests that my material model asks for reaction outputs, but they are not filled, or something like that.